src/main/java/seedu/anilist/logic/parser/AddCommandParser.java

```diff
@@ -25,18 +29,28 @@
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_GENRE);
+                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_EPISODE, PREFIX_STATUS, PREFIX_GENRE);
 
         if (!arePrefixesPresent(argMultimap, PREFIX_NAME)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
+        Episode episode;
+        Status status;
+        if (argMultimap.getValue(PREFIX_EPISODE).isPresent()) {
+            episode = ParserUtil.parseEpisode(argMultimap.getValue(PREFIX_EPISODE).get());
+        } else {
+            episode = new Episode("0");
+        }
+        if (argMultimap.getValue(PREFIX_STATUS).isPresent()) {
+            status = ParserUtil.parseStatus(argMultimap.getValue(PREFIX_STATUS).get());
+        } else {
+            status = new Status("w");
+        }
```

Comment:
Probably should define a default status and default episode, so that we can use it here.